<launch>
  <arg name="robot_name"        default="b_bot"/>
  <arg name="reference_frame"	default="workspace_center"/>
  <arg name="sensor_frame"	default="$(arg robot_name)_ft300_wrench_link"/>
  <arg name="rate"              default="100"/>
  <node name="$(arg robot_name)_ftsensor"
        pkg="aist_ftsensor" type="calibration_server" output="screen">
    <param name="reference_frame"	value="$(arg reference_frame)" />
    <param name="sensor_frame"		value="$(arg sensor_frame)" />
    <param name="rate"			value="$(arg rate)" />
    <remap from="/wrench_in" to="/$(arg robot_name)_controller/wrench"/>
  </node>
</launch>
